testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -634,6 +634,16 @@ def test_boolean_indexing(self):
         for at in [ag[0], ag[2]]:
             assert_equal(at in ag2, True)
 
+    def test_boolean_indexing_2(self):
```

Comment:
I'm not really sure what that decorator does... I think it would allow you to selectively run tests with that tag.  So maybe it's redundant now we're using Travis anyway?

If I had a problem with boolean indexing, I'd probably grep for the tests rather than see if it was an issue once (not all features are issues).